package.json

```diff
@@ -20,6 +20,7 @@
         "eslint-plugin-react": "3.16.1"
     },
     "dependencies": {
+        "@dicetechnology/avdoris": "git+ssh://git@github.com/DiceTechnology/avdoris.git#semver:=1.11.0",
         "@imggaming/dice-shield": "git+ssh://git@github.com:DiceTechnology/dice-shield.git#semver:=2.20.2",
```

Comment:
We should bump the dice-shield to 2.20.8, to match the dice-shield-android 0.19.5 within upgraded exoDoris 2.5.0